### DISCUSSION TITLE
Login via other authentication providers - i.e. google

### DISCUSSION THREAD

**Discussion by lumpi1982** [DarwinsBuddy/WienerNetzeSmartmeter#13] [Q&A] [answered]
To login to the smartmeter portal at https://smartmeter-web.wienernetze.at/ I am using my google account as authentication provider. Do you have any idea how to make this work with this integration?

**Comment by DarwinsBuddy** [comment, marked as answer]
it's not supported right now. You can try to create an account at log.wien to log in and use it for the smartmeter login

**Reply by DarwinsBuddy** [reply]
https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/discussions/19
